Networking/Networking/Requests/WordPressOrgRequest.swift

```diff
@@ -0,0 +1,54 @@
+import Foundation
+import Alamofire
+
+/// Represents a WordPress.org REST API Endpoint
+///
+struct WordPressOrgRequest: URLRequestConvertible {
+
+    /// Base URL for the endpoint
+    ///
+    let baseURL: String
+
+    /// HTTP Request Method
+    ///
+    let method: HTTPMethod
+
+    /// Path to endpoint
+    ///
+    let path: String
+
+    /// Parameters
+    ///
+    let parameters: [String: Any]
```

Comment:
```suggestion
    var parameters: [String: Any] = [:]
```

Can we do this to avoid having a custom `init` method? 🤔

Reply:
Makes sense! Updated in 788471b65b71cc00598046029bea2b523340fa44.